python/xorbits/_mars/dataframe/datasource/read_parquet.py

```diff
@@ -495,6 +520,14 @@ def _pandas_read_parquet(cls, ctx: dict, op: "DataFrameReadParquet"):
         out = op.outputs[0]
         path = op.path
 
+        if op.is_http_url:
+            ctx[out.key] = pd.read_parquet(
+                op.path,
+                columns=op.columns,
+                engine=op.engine,
+                **op.read_kwargs or dict(),
```

Comment:
The `use_arrow_dtype` should be handled here. If `use_arrow_dtype` is True, `pd.read_parquet` use a arrow type mapping.